XenAdmin/Controls/VerticalTabs.cs

```diff
@@ -48,17 +48,32 @@ public interface IVerticalTab
             Image Image { get; }
         }
 
+        private const int DEFAULT_ITEM_HEIGHT = 40;
+
         public VerticalTabs()
         {
-            base.ItemHeight = 40;
+            ItemHeight = DEFAULT_ITEM_HEIGHT;
             IntegralHeight = false; 
             DrawMode = DrawMode.OwnerDrawFixed;
         }
 
-        public override int ItemHeight
+        public sealed override int ItemHeight
         {
             get => base.ItemHeight;
-            set => throw new NotSupportedException($"Invalid set call for '{MethodBase.GetCurrentMethod()?.Name}', value: '{value}'");
+            set
+            {

```

Comment:
Isn't this more complicated than the original code? It even qualifies for a comment.

Reply:
that's true but it uses `value`, which is the original issue brought up by SonarQube.

The only other options are to:
1. Disable the rule
2. Review every instance of the rule being broken in the codebase, and flag it as "won't fix" on SonarQube

Reply:
Why don't we just throw the exception? Plus that, if the idea is not to use the setter, line 55 will cause confusion.

Reply:
Even with this format:
```csharp
public VerticalTabs()
        {
            IntegralHeight = false; 
            DrawMode = DrawMode.OwnerDrawFixed;
        }

        public sealed override int ItemHeight
        {
            get => DEFAULT_ITEM_HEIGHT;
            set => throw new NotSupportedException($"Invalid set call for '{MethodBase.GetCurrentMethod()?.Name}', value: '{value}'");
            
        }
```

The exception is still thrown:
![image](https://user-images.githubusercontent.com/32554698/155490271-da276bab-dfef-4495-b94f-24bb919aed60.png)

(Open `Tools > Options`)

You're right, the line 55 is not necessary since that's the default `ItemHeight`, anyway. I can remove it

Reply:
I think we should just remove setting the virtual members from the constructor and use the a partial override instead:
```
public override int ItemHeight => 40;
public override DrawMode DrawMode => DrawMode.OwnerDrawFixed;
```

Reply:
sounds good to me. Will do shortly